File: Modul 1/PRAK110_2410817310016_ARIFFATAHILLAHPERMANA.py
from math import sqrt

alas = 5
tinggi = 12 

print("diketahui")
print(f"Alas ={alas} cm")
print(f"tinggi ={tinggi} cm")

sisi_miring = sqrt(pow(alas,2) + pow(tinggi,2))
keliling = alas + tinggi + sisi_miring
luas = 1/2 * alas * tinggi

print("Jawab:")
print(f"Sisi A ={tinggi}cm")
print(f"Sisi B ={sisi_miring:.0f}cm")
print(f"Sisi C ={alas}cm")
print(f"keliling = {keliling:.0f}cm")
print(f"luas = {luas:.0f}cm")

File: Modul 1/PRAK110_2410817310016_ARIFFATAHILLAHPERMANA.c
#include <stdio.h>
#include <math.h>

int main()
{
    int alas = 5;
    int tinggi = 12;
    
    printf("diketahui\n");
    printf("Alas =  %d\n", alas);
    printf("Tinggi =  %d\n", tinggi);

    int sisi_miring = sqrt(pow(alas,2) + pow(tinggi,2));
    int keliling = alas + tinggi + sisi_miring;
    int luas =  alas * tinggi / 2;

    printf("jawab\n");
    printf("Sisi A %d\n", tinggi);
    printf("Sisi B %d\n", sisi_miring);
    printf("Sisi C %d\n", alas);
    printf("keliling %d\n", keliling);
    printf("luas %d", luas);
    return 0;
}